### ISSUE TITLE
Deep Learning Traversability Estimator for Mobile Robots in Unstructured Environments. (arXiv:2105.10937v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#7221]
Deep Learning Traversability Estimator for Mobile Robots in Unstructured Environments. (arXiv:2105.10937v1 [cs.RO])<br>
https://ift.tt/34c2L99<br>
<p>Terrain traversability analysis plays a major role in ensuring safe robotic navigation in unstructured environments. However, real-time constraints frequently limit the accuracy of online tests, especially in scenarios where realistic robot-terrain interactions are complex to model. In this context, we propose a deep learning framework, trained in an end-to-end fashion from elevation maps and trajectories, to estimate the occurrence of failure events. The network is first trained and tested in simulation over synthetic maps generated by the OpenSimplex algorithm. The prediction performance of the Deep Learning framework is illustrated by being able to retain over 94% recall of the original simulator at 30% of the computational time. Finally, the network is transferred and tested on real elevation maps collected by the SEEKER consortium during the Martian rover test trial in the Atacama desert in Chile. We show that transferring and fine-tuning of an application-independent pre-trained model retains better performance than training uniquely on scarcely available real data.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
